buildenv

```diff
@@ -0,0 +1,39 @@
+export ZOPEN_GIT_URL="https://gitlab.freedesktop.org/libbsd/libbsd"
+export ZOPEN_GIT_DEPS="libmd git make m4 autoconf perl libtool automake bash zoslib tar sed coreutils gnulib"
+#export ZOPEN_TARBALL_URL="https://libbsd.freedesktop.org/releases/libbsd-0.11.7.tar.xz"
+#export ZOPEN_TARBALL_DEPS="make libmd"
+export ZOPEN_TYPE="GIT"
+export ZOPEN_RUNTIME_DEPS="bash"
+export ZOPEN_COMP="clang"
+export ZOPEN_BOOTSTRAPD="./autogen"
```

Comment:
ZOPEN_BOOTSTRAPD represents the default.

Can you use ZOPEN_BOOTSTRAP instead

Reply:
Done